tests/emails_test.py

```diff
@@ -62,3 +64,20 @@ def test_should_get_email_raise_exception_when_no_content(self) -> None:
             _ = resend.Emails.get(
                 email_id="4ef9a417-02e9-4d39-ad75-9611e0fcc33c",
             )
+
+    def test_email_response_html(self) -> None:
+        self.set_magic_mock_obj(
+            MagicMock(
+                status_code=200,
+                headers={"content-type": "text/html; charset=utf-8"},
+                text="<strong>hello, world!</strong>",
+            )
+        )
+        params: resend.Emails.SendParams = {
+            "to": "to@email.com",
+            "from": "from@email.com",
+            "subject": "subject",
+            "html": "html",
+        }
+        with self.assertRaises(ResendError):
```

Comment:
let's assert for the error message too `Failed to parse Resend API response. Please try again`

Reply:
☝️ this is important to avoid a scenario where we are asserting for 500, but the error "inside" the API is not the one we are trying to simulate in the test